### PR TITLE
Translation for _toctree, Chapter 0, Chapter 1, Chapter 2 and Course Events in Bengali  

### LINKED ISSUE #47: Translation to BENGALI
Hi there 👋

Let's translate the course to `BENGALI` so that the whole community can benefit from this resource 🌎!

## Chapters
I would like to translate Chapters 0 and 1

### PR DESCRIPTION
**Pull request by susnato**
Partially Fixes #47

### CODE REVIEW

**Comment by susnato** [comment]
_toctree.yml and Chapter 0 is done till now, currently working on Chapter 1

**Comment by susnato** [comment]
Hi @MKhalusova, this PR is ready for review!

**Comment by susnato** [comment]
Chapter 2 Added

File: chapters/ben/chapter1/audio_data.mdx [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+এই ধরনের ভিজ্যুয়ালাইজেশন অডিও সিগন্যালের নির্দিষ্ট বৈশিষ্ট্য যেমন কোনো ঘটনার সময় চিহ্নিত করার জন্য, সিগন্যালের সামগ্রিক উচ্চতা এবং অডিওতে উপস্থিত
+কোনো অনিয়ম বা শব্দ কে চিহ্নিত করে ।
+
+একটি অডিও সিগন্যালের জন্য তরঙ্গরূপ প্লট করার জন্য, আমরা একটি python লাইব্রেরি ব্যবহার করতে পারি যার নাম 'librosa':
+
+```bash
+pip install librosa
+```
+
+Let's take an example sound called "trumpet" that comes with the library:
```
Comment (MKhalusova) [inline chapters/ben/chapter1/audio_data.mdx:90]: It looks like you've missed a sentence.

**Review by MKhalusova — COMMENTED** [review]
Thank you so much for all your efforts translating the course! I spotted one sentence that I think you missed, and for some reason, the build preview isn't building your changes.

CC @mishig25 Can you please take a look to see why the preview isn't working for another language? Thanks!

File: chapters/ben/chapter1/audio_data.mdx (hunk 1 above)
Comment (susnato) [reply chapters/ben/chapter1/audio_data.mdx]: fixed it!

File: .github/workflows/build_pr_documentation.yml [hunk 2]
```diff
@@ -17,4 +17,4 @@ jobs:
       package_name: audio-course
       path_to_docs: audio-transformers-course/chapters/
       additional_args: --not_python_module
-      languages: en
\ No newline at end of file
+      languages: en, ben
```
Comment (mishig25) [inline .github/workflows/build_pr_documentation.yml:20]:
```suggestion
      languages: en ben
```

as [here](https://github.com/huggingface/course/blob/main/.github/workflows/build_pr_documentation.yml#L19), we just list them without comma 😊

File: .github/workflows/build_documentation.yml [hunk 3]
```diff
@@ -14,7 +14,7 @@ jobs:
       package_name: audio-course
       path_to_docs: audio-transformers-course/chapters/
       additional_args: --not_python_module
-      languages: en
+      languages: en, ben
```
Comment (mishig25) [inline .github/workflows/build_documentation.yml:17]: as [here](https://github.com/huggingface/course/blob/main/.github/workflows/build_pr_documentation.yml#L19), we just list them without comma 😊

**Comment by mishig25** [comment]
> CC @mishig25 Can you please take a look to see why the preview isn't working for another language? Thanks!

@susnato thanks a lot for the translation! For the language names, we use 2-letter format. Therefore, according to [this table](https://www.loc.gov/standards/iso639-2/php/code_list.php), instead of `ben`, it should be `bn`. After this change, the preview will work 😊

**Comment by MKhalusova** [comment]
@susnato Please address the comments by @mishig25 so that the course is built correctly, and then we'll be able to merge

**Comment by susnato** [comment]
Hi @mishig25 thanks for the suggestion, the PR preview is working now. :hugs:

**Comment by susnato** [comment]
Hi @MKhalusova , I have pushed the changes and the PR is correctly building now!

**Pull request merged**